libs/feature/dataviz/src/lib/chart-view/chart-view.component.stories.ts

```diff
@@ -21,6 +22,7 @@ export default {
       imports: [
         ChartComponent,
         HttpClientModule,
+        OverlayModule,
```

Comment:
I think we should import the UiInputs module here instead, as OverlayModule is an underlying dependency